### DISCUSSION TITLE
vscode integration error

### DISCUSSION THREAD

**Discussion by Mansalek** [eclipse-glsp/glsp#686] [Q&A]
Hello,
wanting to try the integration with vscode , i'am pretty sure I did everything right but I've encountered this strange error (which seems vscode related? I don't exactly know.
The diagram and palette won't show and the error pops after some time 

![vscodee](https://user-images.githubusercontent.com/100411404/174775696-4702ab78-b21f-46a2-ba09-71bbcee573d4.PNG)
sometimes it won't even show that error but this one 
![Capture](https://user-images.githubusercontent.com/100411404/174782025-5458183d-0b5b-483a-9bce-4a3a3cfa3132.PNG)

**Comment by planger** [comment]
@tortmayr Do you have an idea? Thanks!
